seqr/management/commands/update_igv_location.py

```diff
@@ -0,0 +1,26 @@
+from django.core.management.base import BaseCommand, CommandError
+import logging
+from tqdm import tqdm
+
+from seqr.models import IgvSample
+
+logger = logging.getLogger(__name__)
+
+
+class Command(BaseCommand):
+
+    def add_arguments(self, parser):
+        parser.add_argument('old_prefix')
+        parser.add_argument('new_prefix')
+
+    def handle(self, *args, **options):
+        old_prefix = options['old_prefix']
+        new_prefix = options['new_prefix']
+        igv_samples = IgvSample.objects.filter(file_path__startswith=old_prefix)
+        if not igv_samples:
+            raise CommandError(f'No IGV samples found with file prefix "{old_prefix}"')
+        logger.info(f'Updating {len(igv_samples)} IGV samples')
+        for sample in tqdm(igv_samples):
```

Comment:
is it better to use `bulk_update()` or whatever method can push this into a single call to the db rather than hitting the db for each sample?

Reply:
🤦‍♀️ yes, I used an old manage command as the template for this and didn't think too hard about it but that would be a much better approach